staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -431,31 +455,42 @@ type JobStatus struct {
 	// Job is created in the suspended state, this field is not set until the
 	// first time it is resumed. This field is reset every time a Job is resumed
 	// from suspension. It is represented in RFC3339 form and is in UTC.
+	//
+	// Once set, the field can only be removed when the job is suspended.
+	// The field cannot be modified while the job is unsuspended or finished.
```

Comment:
is it possible to suspend finished job?

Reply:
Yes, currently. 

Technically, we could prevent that, but I don't think we want to block spec (the suspend field is in `spec.suspend`) updates based on status. The new validation is meant to ensure the controllers do proper status updates. I guess we can assume modifying spec is outside of controller duties anyway.

Reply:
okay

Reply:
I suppose it should be a possibility, but it does not seem backwards compatible.

Reply:
Right, while potentially such a transition could be reasonable, modifying spec was always legal in job, so I think we should keep it. This is something someone could always do even without managedBy